app/javascript/styles/mastodon/components.scss

```diff
@@ -604,7 +604,7 @@
   line-height: 20px;
   word-wrap: break-word;
   font-weight: 400;
-  overflow: hidden;
+  overflow: visible;
```

Comment:
This is a bad idea because this line was guarding against zalgo text going outside a single toot's area.

Reply:
I understand, but emojis have negative margin, it is difficult to display well in all unknown devices without space between status__info and status__content.
I update this pull request, add 2px space, like suggention 2 of https://github.com/tootsuite/mastodon/pull/5474#issuecomment-349231615 .